### ISSUE TITLE
Built-in `privacy` plugin breaks image alt text encoding

### ISSUE THREAD

**Issue by Guts** [squidfunk/mkdocs-material#5077]
### Context

Adding some image alt text and description with special chars that appear bad encoded, visible to end-reader through glightbox plugin.

### Bug description

Mkdocs.yml :

```yml
site_name: "Test Material image title encoding"

theme:
  name: material
  font: false

plugins:
  - info
  - privacy
```

Input markdown:

```markdown
# Welcome to MkDocs

For full documentation visit [mkdocs.org](https://www.mkdocs.org).

## Commands

* `mkdocs new [dir-name]` - Create a new project.
* `mkdocs serve` - Start the live-reloading docs server.
* `mkdocs build` - Build the documentation site.
* `mkdocs -h` - Print help message and exit.

## Project layout

    mkdocs.yml    # The configuration file.
    docs/
        index.md  # The documentation homepage.
        ...       # Other markdown pages, images and other files.

## Image with title

![Cette capture d'écran est très intéressante](https://squidfunk.github.io/mkdocs-material/assets/screenshots/navigation-tabs.png)
```

Output HTML :

```html
[...]
<h2 id="image-with-title">Image with title</h2>
<p>
  <img
    alt="Cette capture d'Ã©cran est trÃ¨s intÃ©ressante"
    src="assets/external/squidfunk.github.io/mkdocs-material/assets/screenshots/navigation-tabs.png"
  />
</p>
[...]
```

### Related links

- [Built-in privacy](https://squidfunk.github.io/mkdocs-material/setup/ensuring-data-privacy/?h=privacy#built-in-privacy-plugin)
- https://squidfunk.github.io/mkdocs-material/reference/images/?h=glig#lightbox

### Reproduction

[example.zip](https://github.com/squidfunk/mkdocs-material/files/10795693/example.zip)

### Steps to reproduce

1. Add `privacy` to mkdocs.yml
2. Add image with an alt text with special characters.
3. Build & look at the HTML

### Browser

Chrome, Firefox

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting! Fixed in [`e34ad9182`](https://github.com/squidfunk/mkdocs-material-insiders/commit/e34ad9182a6fa04e5bbad8c422fc7c50a54f0ea1).

**Comment by squidfunk** [comment]
Released as part of [9.0.14+insiders-4.32.1](https://github.com/squidfunk/mkdocs-material-insiders/releases/tag/9.0.14-insiders-4.32.1).
